### PR TITLE
Workflows

### PR DESCRIPTION
**Pull request by BartlomiejWojcik**

### CODE REVIEW

File: .github/workflows/master-pr-tests.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      with:
+        node-version: 14.x
+
+    - name: e2e tests
+      env:
+        DEFAULT_BRANCH: master
+        GH_TOKEN: ${{ secrets.GH_TOKEN }}
+      run: |
+        npm install
+        npm run lint
```
Comment (ArturBa) [inline .github/workflows/master-pr-tests.yml:49]: since lint is in the 1. job I don't think it's required here

**Review by ArturBa — CHANGES_REQUESTED** [review]
please remove duplicated linting

File: .github/workflows/master-pr-tests.yml (hunk 1 above)
Comment (BartlomiejWojcik) [reply .github/workflows/master-pr-tests.yml]: Sure, second lint removed

**Review by ArturBa — APPROVED** [review]
nice. thx a lot :D

**Pull request merged**